xen.spec.in

```diff
@@ -236,92 +207,40 @@ BuildRequires: checkpolicy m4
 # cross compiler for building 64-bit hypervisor on ix86
 BuildRequires: gcc-x86_64-linux-gnu
 %endif
-# gcc with BTI mitigation, for dom0 build
-%if 0%{?fedora} == 25
-#BuildRequires: gcc >= 6.4.1-1.qubes1
-%endif
-BuildRequires: bison
-BuildRequires: flex
-Requires: bridge-utils
-Requires: python2-lxml
-Requires: udev >= 059
+BuildRequires: gcc
+Requires: iproute
+Requires: python3-lxml
 Requires: xen-runtime = %{version}-%{release}
 # Not strictly a dependency, but kpartx is by far the most useful tool right
 # now for accessing domU data from within a dom0 so bring it in when the user
 # installs xen.
 Requires: kpartx
-Requires: chkconfig
 ExclusiveArch: %{ix86} x86_64 armv7hl aarch64
 #ExclusiveArch: %#{ix86} x86_64 ia64 noarch
 %if %with_ocaml
 BuildRequires: ocaml, ocaml-findlib
 %endif
-# efi image needs an ld that has -mi386pep option
-%if %efiming
-BuildRequires: mingw64-binutils
-%endif
 %if %with_systemd_presets
 Requires(post): systemd
 Requires(preun): systemd
-Requires(postun): systemd
 BuildRequires: systemd
 %endif
 BuildRequires: systemd-devel
 %ifarch armv7hl aarch64
 BuildRequires: libfdt-devel
 %endif
-# BIOS for HVMs
-%if 0%{?rhel} >= 7
-BuildRequires: OVMF
-%else
+%if %build_ovmf
 BuildRequires: edk2-ovmf
 %endif
-Requires: seabios-bin
 
 %description
 This package contains the XenD daemon and xm command line
 tools, needed to manage virtual machines running under the
 Xen hypervisor
 
-%if %with_python2
-%package -n python2-%{name}
-Summary: Python2 bindings for Xen tools
-Group: Development/Libraries
-Requires: xen-libs = %{version}-%{release}
-Requires: python2
-%{?python_provide:%python_provide python2-%{name}}
-
-%description -n python2-%{name}
-This package contains Python2 bindings to Xen tools. Especially xen.lowlevel.xs
-and xen.lowlevel.xc modules.
-
-%endif
-
-%if %with_python3
-%package -n python%{python3_pkgversion}-%{name}
```

Comment:
When dropping this, `python%{python3_pkgversion}-%{name}` need to be provided by other package that now contains python bindings, otherwise there will be broken dependencies (for example qubes-core-dom0 depends on python3-xen). It looks like it's now shipped in `xen` package (which is IMO weird, if not `python3-xen`, then should be `xen-libs`). Maybe upstream repository/bugtracker contains explanation for this?

Reply:
It looks like it's thanks to RPM: https://fedoraproject.org/wiki/Changes/Automatic_Provides_for_Python_RPM_Packages

Reply:
@marmarek, I think we need to readd this section for python3-xen instead of having it in xen. I was misleaded by the reference link.

Reply:
Personally I think python3-xen package would make more sense. But since it's needed in dom0 only, where `xen` package is installed anyway, it doesn't really matter.

Reply:
What does matter is to provide `python3-xen` name - either via real subpackage, or from the main package.